format/src/main/thrift/schema.thrift

```diff
@@ -124,6 +124,7 @@ struct TableSchema{
 	1: required string table_id;  // ID used to
 	2: required list<ColumnSchema> table_columns; // Columns in the table
 	3: required SchemaEvolution schema_evolution; // History of schema evolution of this table
+	4: optional i32 block_size
```

Comment:
@Zhangshunyu Actually need not change Schema for such extra properties which are being added. We can directly store TABLE_PROPERTIES as a string key value pairs in thrift. So that all such properties configuration parameters can be effectively handled without modifying thrift in future. @jackylk please comment.